infra/nncc/cmake/buildtool/config/config_armv7l-linux.cmake

```diff
@@ -0,0 +1,17 @@
+#
+# armv7l linux compile options
+#
+
+message(STATUS "Building for ARMv7l Linux")
+
+# include linux common
+include("${CMAKE_CURRENT_LIST_DIR}/config_linux.cmake")
+
+# addition for arm-linux
+set(FLAGS_COMMON ${FLAGS_COMMON}
+    "-mcpu=cortex-a7"
+    "-mfloat-abi=hard"
```

Comment:
FYI, tizen recently introduces hf(hard float) architecture named armv7hl. Thus, armv7l in tizen means armv7 with softfp while armv7hl means with hard fp.